src/features/core/components/compound/tel-input.tsx

```
@@ -0,0 +1,98 @@
+import type {
+  Country,
+  DefaultInputComponentProps,
+} from 'react-phone-number-input';
+import enLabels from 'react-phone-number-input/locale/en';
+import koLabels from 'react-phone-number-input/locale/ko';
+import PhoneInputWithCountrySelect from 'react-phone-number-input/react-hook-form';
+
+import { cn } from '@/features/core';
+
+export interface TelInputProps<T extends FieldValues>
+  extends Omit<
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)